AsyncDisplayKit/Private/ASDefaultPlayButton.m

```diff
@@ -0,0 +1,71 @@
+//
+//  ASDefaultPlayButton.m
+//  AsyncDisplayKit
+//
+//  Created by Luke Parham on 1/27/16.
+//  Copyright © 2016 Facebook. All rights reserved.
+//
+
+#import "ASDefaultPlayButton.h"
+
+@implementation ASDefaultPlayButton
+
+- (instancetype)init
+{
+  if (!(self = [super init])) {
+    return nil;
+  }
+  
+  self.opaque = NO;
+  
+  return self;
+}
+
++ (void)drawRect:(CGRect)bounds withParameters:(id<NSObject>)parameters isCancelled:(asdisplaynode_iscancelled_block_t)isCancelledBlock isRasterizing:(BOOL)isRasterizing
+{
+  CGFloat originX = bounds.size.width/4;
+  CGRect buttonBounds = CGRectMake(originX, bounds.size.height/4, bounds.size.width/2, bounds.size.height/2);
+  CGFloat widthHeight = buttonBounds.size.width;
+
+  if (bounds.size.width < bounds.size.height) {
+    //then use the width to determine the rect size then calculate the origin x y
+    widthHeight = bounds.size.width/2;
+    originX = (bounds.size.width - widthHeight)/2;
+    buttonBounds = CGRectMake(originX, (bounds.size.height - widthHeight)/2, widthHeight, widthHeight);
+  }
+  if (bounds.size.width > bounds.size.height) {
+    //use the height
+    widthHeight = bounds.size.height/2;
+    originX = (bounds.size.width - widthHeight)/2;
+    buttonBounds = CGRectMake(originX, (bounds.size.height - widthHeight)/2, widthHeight, widthHeight);
+  }
+  
+  if (!isRasterizing) {
+    [[UIColor clearColor] set];
+    UIRectFill(bounds);
```

Comment:
This has no effect because clear color is completely transparent, correct? What is the intent here, maybe there is a better way?